x/auth/tx/msg_service.go

```diff
@@ -0,0 +1,31 @@
+package tx
+
+import (
+	gocontext "context"
+	"fmt"
+
+	"google.golang.org/grpc"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// Invoke implements the grpc ClientConn.Invoke method.
+func (w *wrapper) Invoke(_ gocontext.Context, method string, args, reply interface{}, _ ...grpc.CallOption) error {
+	req, ok := args.(sdk.MsgRequest)
```

Comment:
can we force this type on the arguments list?
```
Invoke(_ gocontext.Context, method string, args msgRequest, reply interface{}, _ ...grpc.CallOption)
```

Reply:
OK, we can't because of the `ClientConn` interface.

Let's add here a comment pointing to the StdBuilder or just explaning why we need this method.

Reply:
> pointing to the StdBuilder

I'm not sure I follow.

Reply:
I added a comment.

Reply:
This is good. Thanks.